modules/bridge/src/main/ts/ephox/bridge/components/menu/FancyMenuItem.ts

```diff
@@ -23,6 +25,7 @@ const fancyTypes: (keyof FancyActionArgsMap)[] = [ 'inserttable', 'colorswatch'
 export const fancyMenuItemSchema = StructureSchema.objOf([
   FieldSchema.requiredString('type'),
   FieldSchema.requiredStringEnum('fancytype', fancyTypes),
+  FieldSchema.defaulted('initData', {}),
```

Comment:
Could we maybe name this `auxData` or something like that? Normally `initData` or `initialData` implies that the data is mutable, and what we're passing in is just the starting / seed value. In this case it looks like we're just passing in "something extra that gets interpreted by silver in a super specific way depending on what your UI is". This is not necessarily a bad thing, because it's a super custom component, but I'd rather `aux` or `extra` as a name personally

Reply:
That's not how this is working? It is immutable data that defines how the color swatch should render.